searx/engines/packagist.py

```diff
@@ -0,0 +1,78 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# lint: pylint
+# pylint: disable=missing-function-docstring
+"""Packagist (IT)
+
+"""
+
+from json import loads
+from urllib.parse import urlencode
+from dateutil import parser
+
+import requests
```

Comment:
Can you replace by `import searx.network`

And then line 38, replace 
`requests.get(package_url + ".json")`
by
`searx.network.get(package_url + ".json")`

So it uses the same HTTP connection.

Even more be simplified:
```python
body = searx.network.get(package_url + ".json").json()
```

So there is no need to import `from json import loads`.

Reply:
> Can you replace by

See e3c1286 

> So there is no need to import `from json import loads`.

Nit: still used in the response function  / `body = loads(resp.text)`